gulpfile.js

```diff
@@ -49,6 +50,11 @@ gulp.task('raml-lint', () => {
         .pipe(eslint.failAfterError());
 });
 
-gulp.task('ci-checks', ['js-lint', 'json-lint', 'xml-lint', 'validate-naming']);
+gulp.task('unittest', function () {
+  return gulp.src(['**/js/tests/unitTests.js', NO_MODULES])
+    .pipe(ava({verbose: true}))
+});
+
+gulp.task('ci-checks', ['js-lint', 'json-lint', 'xml-lint', 'validate-naming', 'unittest']);
```

Comment:
>js-lint', 'json-l [](start = 25, length = 17)

Do you know if js-lint will check for spacing issues, style issues (eg enclosing if blocks in braces etc, position of braces like the ts-linter we use for opent2t repo?